client/src/pages/container/DashboardDefault.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Container, Typography } from "@material-ui/core";
 import FriendListCard from "../presentational/FriendListCard";
-import FriendListDialog from "../presentational/FriendListDialog";
+import FriendListDialog from "../presentational/FriendlistDialog";
```

Comment:
I changed this one to `FriendListDialog` so we can be more consistent in naming thru out the app (with a capital `L`)

Reply:
the actual component has a lower case l that's why i changed it i'll just change the component name